### ISSUE TITLE
Waffle Framework:Login pop up on session timeout in internet explorer on windows 7/8 Issue.

### ISSUE THREAD

**Issue by nttits** [dblock/waffle#199]
I am facing with the problem in my web application where I am asked to enter my login credentials again on session time out.This issue arises on Windows7 and Windows 8 Internet Explorer only.
Ideally what I want is that the pop up for credentials should not arise and user should be automatically logged in after session time out .

Note: 
1>Single Sign-On works well with other browsers like Firefox on Windows 7 and 8 as well(No login pop up issues here).
2>It also works fine with Internet Explorer on Windows Xp. 
3>My web application uses Weblogic server 10.3, waffle-jna jar version 1.5 and I am running it on Internet Explorer 11 on Windows 8.
4>I have tried with the latest versions of Waffle (ver 1.7.3 and ver 1.6) in my web app but still I am facing the same problem.

To sum up ,  On windows 7 and windows 8  with the browser Internet explorer Single Sign-On fails and I am asked to enter the login credentials on session time out.Is this the issue with Waffle framework? Please provide any help.

**Comment by dblock** [comment]
Bring this to the mailing list, please.

**Comment by nttits** [comment]
Hi, I am new to github . Can you please elaborate as to what I need to do to "Bring this to the mailing list" ?

**Comment by dblock** [comment]
This is likely not a bug, there's a [Google Group / Mailing List](https://groups.google.com/forum/#!forum/waffle-users) linked under "Need Help" in the README, where you can ask questions.
